lib/charms/data_platform_libs/v0/object_storage.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python3
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import logging
+from collections import namedtuple
+from typing import Dict, List, Optional
+
+from charms.data_platform_libs.v0.data_interfaces import (
+    EventHandlers,
+    ProviderData,
+    RequirerData,
+    RequirerEventHandlers,
+)
```

Comment:
We agree that this is a temporary solution and once the data_interfaces lib is published as separate Python repository, we will rethink about where the object storage integrator lib code will reside and how it will inherit the data_interfaces code.